exercises/rotational-cipher/src/test/java/RotationalCipherTest.java

```diff
@@ -0,0 +1,80 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class RotationalCipherTest {
+    private  RotationalCipher rotationalCipher;
+
+    @Before
+    public void setUp() {
+        rotationalCipher = new RotationalCipher();
+    }
+
+    @Test
+    public void rotateEmpty() {
+        Assert.assertEquals("", rotationalCipher.rotate("", 13));
+    }
+
+
+    @Ignore("Remove to run test")
+    @Test
+    public void rotateLowerCaseWithoutSpaces() {
+        Assert.assertEquals("nopqrstuvwxyzabcdefghijklm", rotationalCipher.rotate("abcdefghijklmnopqrstuvwxyz", 13));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void rotateBy5() {
+        Assert.assertEquals("trl", rotationalCipher.rotate("omg", 5));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void rotateLowerCaseWithSpaces(){
+        Assert.assertEquals("nopqr stuvw xyzab cdefg hijkl m", rotationalCipher.rotate("abcde fghij klmno pqrst uvwxy z", 13));
```

Comment:
For very long lines it makes it more readable to have each argument on a new line, e.g.
```
functionName(
  argument1,
  argument2
);
```
:)